src/components/DonutChart/DonutChart.stories.mdx

```diff
@@ -0,0 +1,45 @@
+# DonutChart Component
+
+## Type: Simple
+
+## DonutChart Props
+
+|        Prop        |  Type  |                                                      Description                                                       | Default | Mandatory |
```

Comment:
@branquinhoaa is it possible to recreate exactly the donut in the picture on https://github.com/codelittinc/agnostic-design-system/issues/306 with these props? I'm not seeing how we would add the `risk` message an icon there

Reply:
In the issue I make a suggestion about how we could achieve it